generators/typescript/sdk/client-class-generator/src/websocket/GeneratedDefaultWebsocketImplementation.ts

```diff
@@ -176,7 +176,8 @@ export class GeneratedDefaultWebsocketImplementation implements GeneratedWebsock
                     const typeNode = queryParameter.allowMultiple
                         ? ts.factory.createUnionTypeNode([
                               type.typeNodeWithoutUndefined,
-                              ts.factory.createArrayTypeNode(type.typeNodeWithoutUndefined)
+                              ts.factory.createArrayTypeNode(type.typeNodeWithoutUndefined),
+                              ts.factory.createKeywordTypeNode(ts.SyntaxKind.UndefinedKeyword)
```

Comment:
um I guess this works? Should we instead be keying off of whether this allows undefined?